sandy/core/cov.py

```diff
@@ -2179,11 +2193,11 @@ def segmented_pivot_table(data_stack, rows = 10000000,
     # disappear, keeping the original values.
     pivot_matrix = pivot_matrix.groupby(pivot_matrix.index).sum()
     if len(index) >= 2:
-        # Groupby transform multiindex structure into a tuple. This line
+        # Groupby transforms multiindex structure into a tuple. This line
         # reverse the transformation.
         pivot_matrix.index = pd.MultiIndex.from_tuples(
             pivot_matrix.index,
-            name=index,
+            names=index,

```

Comment:
you changed this...but you don't have any test showing how it works

Reply:
should index and columns have default values or not?

Reply:
IT was a mistake, the real argument is names, so my last pull request fails because the multi index options was bad written

Reply:
I will erase the default values